widgets/testimonial/styles/default.less

```diff
@@ -1,6 +1,5 @@
-@import "../../../base/less/mixins";
```

Comment:
I think this is automatically handled by preprocessing now. Although the only problem is that it'll no longer autocomplete the mixins in something like PHP Storm.